src/components/avatar/index.tsx

```diff
@@ -222,14 +222,21 @@ class Avatar extends PureComponent<AvatarPropTypes> {
     return _.get(this.props, 'badgeProps.backgroundColor') || statusColor || onlineColor;
   }
 
-  getBadgeSize = () => _.get(this.props, 'badgeProps.size', DEFAULT_BADGE_SIZE);
```

Comment:
Same here, I think your PRs got mixed up
make sure when create a new branch to always create it out of `master`

Reply:
sure, fixed